compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeWindowDelegate.desktop.kt

```diff
@@ -143,8 +148,40 @@ internal class ComposeWindowDelegate(
                 LocalWindow provides window,
                 LocalLayerContainer provides _pane
             ) {
-                content()
-                undecoratedWindowResizer.Content()
+                val density = LocalDensity.current.density
+                Layout(
+                    {
+                        content()
+                        undecoratedWindowResizer.Content()
+                    },
+                    Modifier,
+                    measurePolicy = { measurables, constraints ->
+                        val contentMeasurable = measurables[0]
+                        val resizerMeasurable = measurables.getOrNull(1)
+
+                        val contentPlaceable = contentMeasurable.measure(constraints)
+                        val width: Int = max(constraints.minWidth, contentPlaceable.width)
+                        val height: Int = max(constraints.minHeight, contentPlaceable.height)
+
+                        val resizerPlaceable = resizerMeasurable?.let{
+                            val resizerWidth = (window.width * density).toInt()
+                            val resizerHeight = (window.height * density).toInt()
+                            it.measure(
+                                Constraints(
+                                    minWidth = resizerWidth,
+                                    minHeight = resizerHeight,
+                                    maxWidth = resizerWidth,
+                                    maxHeight = resizerHeight
+                                )
+                            )
+                        }
+
+                        layout(width, height){
```

Comment:
Is it possible to use `constraints.maxWidth`? It is the case before this PR (see RootMeasurePolicy)

Reply:
The incoming constraints are `screenSize.width x screenSize.height`, so this makes the window to always have the size of the screen (even decorated).
I'm not sure why it does work with `RootMeasurePolicy`. The size of the window is set to the value returned by `SkiaBasedOwner.computeContentSize`:
```
    // Don't use mainOwner.root.width here, as it strictly coerced by [constraints]
    private fun computeContentSize() = IntSize(
        root.children.maxOfOrNull { it.outerCoordinator.measuredWidth } ?: 0,
        root.children.maxOfOrNull { it.outerCoordinator.measuredHeight } ?: 0,
    )
```

Reply:
It worked before, because RootMeasurePolicy measures the root node. Now it is not the root node. So yes, we can't have this.

Having the `Box` logic as it is now sounds reasonable.